CMakeLists.txt

```
@@ -3,7 +3,7 @@
 cmake_minimum_required(VERSION 3.15)
 
 # sccache cannot handle the -Fd option generationg pdb files
-if ("MSVC" STREQUAL "${CMAKE_CXX_COMPILER_ID}")
+if ("MSVC" STREQUAL "${CMAKE_CXX_COMPILER_ID}" AND CMAKE_VERSION VERSION_GREATER_EQUAL 3.25)
   cmake_policy(SET CMP0141 NEW)
 endif()
```

Question: Why cannot we write:
```suggestion
if ("MSVC" STREQUAL "${CMAKE_CXX_COMPILER_ID}" AND POLICY CMP0141)
  cmake_policy(SET CMP0141 NEW)
endif()
```
CMake has a conditional to check if a policy exists: https://cmake.org/cmake/help/latest/command/if.html#policy

---

Nice catch! I didn't know about this one.

---

Also dropping the compiler check since this policy only affects MSVC.

---

Great!